### ISSUE TITLE
Request: Read and Post Group Chat Messages

### ISSUE THREAD

**Issue by Hennamann** [Bungie-net/api#137]
Could we perhaps get API functionality for clan chat messages, both for looking at existing messages in the chat and for posting new messages to the chat, programmatically?

**Comment by vthornheart-bng** [comment]
I'll ask around about it - we have an API for it but we haven't marked it as "public" so there's not really any documentation to be had.  I'll ask about whether it'd be kosher to go ahead and mark it as public.

**Comment by maciek134** [comment]
Can we use the api that the clan page uses, before there is documentation? I didn't check with a logged in user, but the endpoint responds when called from CURL.

**Comment by Hennamann** [comment]
Yeah, I found those endpoints as well, namely the "Message/GetConversationThreadV3" endpoint.
But even with Oauth, it returns an Access not permitted by application scope error. As my application does not have the ReadGroups scope, which is seemingly not available outside of Bungie yet, along with a lot of other API scopes.
